pkg/dispatcher/dispatcher_test.go

```diff
@@ -201,31 +204,57 @@ func TestDispatchTimeout(t *testing.T) {
 	}()
 
 	// pretend to be an time-consuming event-handler
```

Comment:
This comment does not make sense any more as the sleep has been removed

Reply:
Fixed.